x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_types/utils/log_cluster_shard_failures_esql.test.ts

```diff
@@ -42,7 +42,7 @@ describe('logClusterShardFailuresEsql', () => {
   it('should add warning message when shard failures exist in a single cluster', () => {
     const shardFailure: EsqlEsqlShardFailure = {
       reason: { type: 'test_failure', reason: 'test failure' },
-      shard: '0',
+      shard: 0,
```

Comment:
This is also correct: https://github.com/elastic/elasticsearch-specification/pull/4515